aspnetcore/fundamentals/middleware/write.md

```diff
@@ -34,15 +35,15 @@ The middleware class must include:
   * Return a `Task`.
   * Accept a first parameter of type <xref:Microsoft.AspNetCore.Http.HttpContext>.
   
-Any additional constructor and `Invoke`/`InvokeAsync` parameters will be populated by [dependency injection (DI)](xref:fundamentals/dependency-injection).
+Additional constructor and `Invoke`/`InvokeAsync` parameters are populated by [dependency injection (DI)](xref:fundamentals/dependency-injection).
```

Comment:
pluralize "constructor" ...

```suggestion
Additional constructors and `Invoke`/`InvokeAsync` parameters are populated by [dependency injection (DI)](xref:fundamentals/dependency-injection).
```

Reply:
@guardrex It's not really meant to be additional constructors. I realise you did suggest this first time through, but I'm trying to state that parameters for the constructor already mentioned and for `Invoke`/`InvokeAsync` are taken from DI. There's no reason to have more than one constructor for the middleware class. Sorry if I'm still not explaining that well enough...

Reply:
I see. No ... I see what you mean. It was just that when you had "any" I didn't realize that you only meant one.

Yes on that :point_up: change. That looks great.

Reply:
Ok, great. I'll make that change now.

Thanks again for this prompt service you're providing. :laughing:

Reply:
Prompt and **_almost accurate_** review services! ... that's what we do. :smile: